Amplifier_Operational.dcm

```diff
@@ -1356,6 +1356,18 @@ K single opamp wideband low-power
 F http://www.ti.com/lit/ds/symlink/opa847.pdf
 $ENDCMP
 #
+$CMP OPA855
+D 8-GHz Gain Bandwidth Product, Gain of 7-V/V Stable, Bipolar Input Amplifier, WSON-8
+K opamp, bipolar
```

Comment:
Please remove commas from keywords, add VFA.
Bipolar is not needed as it is already in the description.